css/styles.css

```diff
@@ -0,0 +1,106 @@
+html {
+  font-size: 62.5%; /* Now 10px = 1rem! */
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+/** GLOBALS **/
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+h1,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+/** UTILITIES **/
+.container {
+  width: 95%;
+  margin: 0 auto;
+}
+
+/** HEADER **/
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 2rem;
+}
+
+.header-heading {
+  color: #6000ec;
+  margin-left: 5rem;
+  font-size: 3rem;
+  text-transform: capitalize;
+}
+
+.header-icon {
+  margin-right: 5rem;
+  font-size: 4rem;
+  color: #6000ec;
+  padding: 1rem;
+  transition: 0.5s ease;
+}
```

Comment:
- [ ] Great job styling your portfolio, I would recommend using the colors provided in the Figma design as required in the project.